modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneAwaitDataNodesTest.java

```diff
@@ -367,20 +377,30 @@ void testRemoveZoneWhileAwaitingDataNodes() throws Exception {
 
         CompletableFuture<Set<String>> dataNodesFut1 = distributionZoneManager.topologyVersionedDataNodes(zoneId, 106);
 
+        doAnswer(invocation -> {
```

Comment:
What are we trying to achieve with this mock? Why it's not enough to simply remove zone, meaning just call
`distributionZoneManager.dropZone(zoneName).get();`
By the way if I do this, test might fail because instead of expected `DistributionZoneWasRemovedException` `DistributionZoneNotFoundException` will be thrown from `org.apache.ignite.internal.distributionzones.DistributionZonesUtil#getZoneById`. We should also fix that.